Scripts/LevelManager.gd

```diff
@@ -92,6 +94,10 @@ func show_offline_screen():
 	var gos = Offline_screen.instance()
```

Comment:
Name gos is acronim of GameOverScreen, so it has no sens in this case. Maybe `offline_screen`, or `os` will be better?

Reply:
Yep, I did not even know, I quickly copied the method from above ;)